scene/gui/text_edit.h

```diff
@@ -1057,6 +1059,12 @@ class TextEdit : public Control {
 	void set_highlight_all_occurrences(const bool p_enabled);
 	bool is_highlight_all_occurrences_enabled() const;
 
+	void set_use_custom_separators(const bool p_enabled);
+	bool is_use_custom_separators_enabled() const;
```

Comment:
```suggestion
	bool is_using_custom_separators_enabled() const;
```
Grammar used elsewhere here (like `is_drawing_spaces`)